packages/scoutgame-ui/src/components/welcome/how-it-works/HowItWorksContent.tsx

```diff
@@ -1,14 +1,18 @@
 'use client';
 
 import { Button, List, ListItem, ListItemAvatar, Stack, Typography } from '@mui/material';
+import { getPlatform } from '@packages/mixpanel/platform';
 import { PointsIcon } from '@packages/scoutgame-ui/components/common/Icons';
 import { useMdScreen } from '@packages/scoutgame-ui/hooks/useMediaScreens';
 import Link from 'next/link';
 import React from 'react';
 
 export function HowItWorksContent({ onClickContinue }: { onClickContinue?: React.MouseEventHandler }) {
   const isMdScreen = useMdScreen();
+  const platform = getPlatform();
   const iconSize = isMdScreen ? 24 : 18;
+  const url = platform === 'telegram' ? '/quests' : '/builders-you-know';
```

Comment:
We should remove this check. /builders-you-know already is responsible to do a redirect if you have no farcaster friends, and it also checks if you have farcaster

Reply:
Done